packages/locale-assets/package.json

```
@@ -0,0 +1,42 @@
+{
+	"name": "@misskey/locale-assets",
```

```suggestion
	"name": "locales",
```

他のpackageはscope切ってないので
後名前長いので`locales`とかのほうが良いかも